scripts/genpotfile

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+usage() { echo "Usage: $0 [-d <po_directory>] <yaml_files>" 1>&2; exit 1; }
+
+while getopts ":d:" opt; do
+    case "${opt}" in
+    d)
+        d=${OPTARG}
+        ;;
+    *)
+        usage
+        ;;
+    esac
+done;
+shift $((OPTIND-1))
+
+if [ -z "${d}" ]; then
+    usage
+fi
+
+toplevel_dir=$(dirname $(readlink -f "$0"))/..
+tmp_dir="/tmp/eos-knowledge-lib/pot"
+po_dir=${d}
+mkdir -p $tmp_dir
+mkdir -p $po_dir
+
+for yaml in ${@:1}; do
+    root=$(basename $yaml | cut -d'.' -f1);
+    $toplevel_dir/tools/autobahn $yaml -c -o $tmp_dir/$root.pot;
```

Comment:
You can just name the file `$yaml.pot` and not bother removing the extension, because you are going to delete the temporary pot file anyway.